### ISSUE TITLE
Temp. doesn't always work?

### ISSUE THREAD

**Issue by rlambert27** [MattHirschfelt/Nest-Extended#13]
Hi Matt,

Check out: http://mynestreport.com/

Do you know why the temperature doesn't always record? If you check June 15th for example, it seems like it doesn't get the temp properly.

Maybe there should be a catch in there if it doesn't get a correct response to use the last temp recorded or simply not add a data point?

Roch

**Comment by amaiman** [comment]
I noticed the same thing around that date (and the day after), it was probably a problem with the Weather Underground API which returned bad data for a few minutes.  I fixed my graphs by going in the SQL database and removing the records with the bad data.  This eventually made a mess of my graphs because the code assumes the SQL datapoints will already be in time order.  I fixed it by adding an ORDER BY statement to the SQL query; I'll submit a pull request with that shortly.
